src/index.js

```diff
@@ -109,7 +110,10 @@ function config (env, _trigger) {
         url: 'https://api.honeycomb.io/v1/traces',
         headers: { 'x-honeycomb-team': env.HONEYCOMB_API_KEY }
       },
-      service: { name: 'freeway' }
+      service: { name: 'freeway' },
+      sampling: {
+        headSampler: new TraceIdRatioBasedSampler(parseFloat(env.FF_TELEMETRY_RATIO))
+      }
```

Comment:
Only enable sampling if ratio is set?

```suggestion
      ...(env.FF_TELEMETRY_RATIO ? {
        sampling: {
          headSampler: new TraceIdRatioBasedSampler(parseFloat(env.FF_TELEMETRY_RATIO))
        }
      } : {})
```